prometheus_s3.tf

```diff
@@ -3,6 +3,8 @@ data template_file "prometheus" {
   vars = {
     parent_domain_name = var.parent_domain_name
     environment        = local.environment
+    nifi_endpoint      = local.nifi_endpoint_url
+    blackbox_hostname  = "${aws_service_discovery_service.blackbox[0].name}.${aws_service_discovery_private_dns_namespace.sdx_services[0].name}"
```

Comment:
Presume the `sdx_services` bit should be `blackbox` here?

Reply:
No. This is a dns namespace in sdx